bindings/cs/rl.net.native/rl.net.live_model.cc

```diff
@@ -136,7 +136,7 @@ API int LiveModelRequestMultiSlotDecision(livemodel_context_t* context, const ch
   RL_IGNORE_DEPRECATED_USAGE_END
 }
 
-API int LiveModelRequestMultiSlotDecisionWithFlags(livemodel_context_t* context, const char* event_id, const char* context_json, unsigned int flags, reinforcement_learning::multi_slot_response* resp, reinforcement_learning::api_status* status)
+API int LiveModelRequestMultiSlotDecisionWithFlags(livemodel_context_t* context, const char * event_id, const char * context_json, unsigned int flags, reinforcement_learning::multi_slot_response_detailed* resp, reinforcement_learning::api_status* status)
```

Comment:
this should not be changed here right? this is still the request for multi slot, not multi slot detailed which was added below

Reply:
must have changed that by mistake, reverted now